astropy/coordinates/tests/test_sky_coord_velocities.py

```diff
@@ -148,6 +148,7 @@ def test_useful_error_missing():
 
 # ----------------------Operations on SkyCoords w/ velocities-------------------
 
+
```

Comment:
Bit weird this, but I guess fine.

Reply:
I think this is because of how # of lines are counted. Ruff may count them a bit differently than flake8. I think it's a small enough issue to not impact merging this PR, but we might bring it to Ruff's attention for a future adjustment.